server/activate.go

```diff
@@ -56,6 +47,16 @@ func (p *Plugin) OnActivate() error {
 	// On Cloud installations we want calls enabled in all channels so we
 	// override it since the plugin's default is now false.
 	if isCloud(p.pluginAPI.System.GetLicense()) {
+		cfg.MaxCallParticipants = new(int)
+		*cfg.MaxCallParticipants = cloudMaxParticipantsDefault
+		if maxPart := os.Getenv("MM_CALLS_MAX_PARTICIPANTS"); maxPart != "" {
```

Comment:
Discovered in QA: Do we not want to allow on-prem to use the `MM_CALLS_MAX_PARTICIPANTS` flag?

Reply:
I think it's more like we need it on Cloud, not so much on on-prem although given it's not possible to use common configuration overrides due to the plugin id issue I think it makes sense to allow this to be used on non-cloud installations as well. I can update :+1: